bin/configs/typescript-consolidated-nullable-enum.yaml

```diff
@@ -0,0 +1,4 @@
+generatorName: typescript
+outputDir: samples/openapi3/client/petstore/typescript/builds/nullable-enum
```

Comment:
for new TS sample folder, do we need to add it to the Circle CI script to ensure the output compiles without issues?

https://github.com/OpenAPITools/openapi-generator/blob/master/CI/circle_parallel.sh#L76

Reply:
I guess, yes, would it make sense to determine this: https://github.com/OpenAPITools/openapi-generator/blob/5732e2758fa3a2bd854fbd12016b49bd5f6ed6ba/CI/circle_parallel.sh#L75-L104

automatically?

Reply:
added bbe6d34 for now

Reply:
not against automating anything

maybe we should take this opportunity to move some tests (which don't require a local petstore server) to github workflows instead

cc @macjohnny